source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -135,7 +181,8 @@ SplitRequestPtr EvalRequest::create(Router& router, Common::Redis::RespValuePtr&
     return nullptr;
   }
 
-  std::unique_ptr<EvalRequest> request_ptr{new EvalRequest(callbacks, command_stats, time_source)};
+  std::unique_ptr<EvalRequest> request_ptr{
+      new EvalRequest(callbacks, command_stats, time_source, delay_command_latency)};
```

Comment:
While you are here just switch all of these to `auto var = std::make_unique<...`

Reply:
That requires me to make the constructors public (as they are private now) or use something like absl::wrapunique, unless I want to do some more messy stuff per what I've read on stackoverflow. Any preference?

Reply:
Don't worry about it.